src/main/java/controller/GameController.java

```diff
@@ -0,0 +1,87 @@
+package controller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import domain.BettingMoney;
+import domain.Name;
+import domain.card.CardDeck;
+import domain.card.CardDeckFactory;
+import domain.participant.Dealer;
+import domain.participant.Player;
+import domain.result.ParticipantProfit;
+import service.DrawService;
+import service.PlayerService;
+import service.ProfitService;
+import view.InputView;
+import view.OutputView;
+
+public class GameController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final PlayerService playerService;
+    private final DrawService drawService;
+    private final ProfitService profitService;
+
+    public GameController(final InputView inputView, final OutputView outputView, final PlayerService playerService,
+            final DrawService drawService, final ProfitService profitService) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.playerService = playerService;
+        this.drawService = drawService;
+        this.profitService = profitService;
+    }
+
+    public void run() {
+        CardDeck cardDeck = CardDeckFactory.createRandomCardDeck();
+        Dealer dealer = new Dealer();
+        List<Player> players = createPlayers();
+
+        drawCards(cardDeck, dealer, players);
+
+        calculateProfit(dealer, players);
+    }
+
+    private List<Player> createPlayers() {
+        String namesWithComma = inputView.inputNames();
+        List<Name> names = Name.fromComma(namesWithComma);
+        List<BettingMoney> bettingMonies = new ArrayList<>();
+        for (final Name name : names) {
+            int money = inputView.inputBettingMoney(name);
+            BettingMoney bettingMoney = new BettingMoney(money);
+            bettingMonies.add(bettingMoney);
+        }
+        return playerService.createPlayers(names, bettingMonies);
+    }
+
+    private void drawCards(final CardDeck cardDeck, final Dealer dealer, final List<Player> players) {
+        drawService.drawInitialCards(cardDeck, dealer, players);
+        outputView.printHandsAtFirst(dealer, players);
+
+        for (final Player player : players) {
+            drawCardToPlayer(cardDeck, player);
+        }
+        drawCardToDealer(cardDeck, dealer);
+    }
```

Comment:
```suggestion
        for (final Player player : players) {
            drawCardToPlayer(cardDeck, player);
        }
    }
```
AllPlayers라는 객체가 있으면, 상속을 극대화하여 사용할 수 있을 것 같은데요
어려울까요..?